specifications/transfer-claim/transfer-claim.yml

```diff
@@ -633,6 +662,67 @@ components:
                     description: In ISO 4217 format
                     example: SEK
 
+    settlement:
+      description: Successful operation
+      content:
+        application/json:
+          schema:
+            type: object
+            required:
+              - financer
+              - currency
+              - createdAt
+              - closedAt
+              - payoutReference
+              - transactions
+            properties:
+              financer:
+                description: Name that should appear on the bank transaction received by the creditor.
+                type: string
+                example: Moank AB
+              currency:
+                  type: string
+                  description: Currency of the payment received by the creditor in ISO 4217 format
+                  example: "SEK"
+              createdAt:
+                description: Date the transactions started to accumulate towards the settlement
+                type: string
+                example: '2023-04-28T12:18:06.849Z'
+                format: date
+              closedAt:
+                description: Date the payout was issued
+                type: string
+                example: '2023-04-28T12:18:06.849Z'
+                format: date
+              payoutReference:
+                description: The payout reference
+                type: string
+                example: 'A fancy reference'
+              transactions:
+                type: array
+                description: Transactions that lead to the settlement
+                items:
+                  type: object
+                  required:
+                    - amount
+                    - date
+                    - description
+                  properties:
+                    amount:
+                      type: number
+                      description: Amount of the transaction
+                      example: '10000'
+                    date:
+                      description: Date the transaction
+                      type: string
+                      example: '2023-04-28T12:18:06.849Z'
+                      format: date
+                    description:
+                      description: Details explanation of the reason for the transaction
```

Comment:
Maybe here we can be a little bit more specific saying that the description include the reference to the financement the transaction is connected to. If we want to go even further, we can explain the difference between the two type of transactions we have: "Amortising financement" and "Financing amount" :thinking:

Reply:
It will be hard to maintain if in the future there are different descriptions or if we update the logic anyhow :thinking:

Reply:
actually those are the two main descriptions we have since the beginning :thinking: 
But then if you change the example to be the current description of a financing transaction, I'm happy with it ;) 
`Order Reference ID: 123 - Financing amount`

Reply:
Ah dang!
I just realised the transaction descriptions in Fimento are talking about orders and we never use that term in hyperfactoring :sweat_smile: 
Not an issue for now but it could be confusing :see_no_evil: